docs/kms_rest_cli.md

```diff
@@ -29,11 +31,11 @@ Parameters can be set by command line arguments or environment variables:
     --kms-secrets-database-url string         The URL of the database for KMS secrets. Not needed if using in-memory storage. For CouchDB, include the username:password@ text if required. Alternatively, this can be set with the following environment variable: KMS_SECRETS_DATABASE_URL
     --kms-secrets-database-prefix string      An optional prefix to be used when creating and retrieving the underlying KMS secrets database. Alternatively, this can be set with the following environment variable: KMS_SECRETS_DATABASE_PREFIX
 
-    --kms-secrets-master-key-path string      The path to the file with master key to be used for secret lock. If missing noop service lock is used. Alternatively, this can be set with the following environment variable: KMS_SECRETS_MASTER_KEY_PATH
-
     --key-manager-storage-type string         The type of storage to use for user's key manager. Supported options: mem, couchdb, edv. Alternatively, this can be set with the following environment variable: KMS_KEY_MANAGER_STORAGE_TYPE
     --key-manager-storage-url string          The URL of storage for user's key manager. Not needed if using in-memory storage. For CouchDB, include the username:password@ text if required. Alternatively, this can be set with the following environment variable: KMS_KEY_MANAGER_STORAGE_URL
     --key-manager-storage-prefix string       An optional prefix to be used when creating and retrieving the underlying user's key manager storage. Alternatively, this can be set with the following environment variable: KMS_KEY_MANAGER_STORAGE_PREFIX
+
+    --hub-auth-url string                     The URL of Hub Auth server to use for fetching secret share for secret lock. If not specified secret lock based on master key is used. Alternatively, this can be set with the following environment variable: KMS_HUB_AUTH_URL
```

Comment:
missing the token configuration for this API

Reply:
it's below

Reply:
in separate section as this is considered to be temporary

Reply:
https://github.com/trustbloc/hub-auth/blob/48d4c6f991b8572826803caa92175e109cd4a71e/pkg/restapi/operation/operations.go#L621-L633

Reply:
sorry, now I see that this is about documentation - do you suggest adding description for that parameter? (I didn't add it because of temporary status)

Reply:
Ok - I see the configuration for it was added but we forgot to update these docs.

Reply:
@aholovko it's fine